src/lib/viewers/controls/annotations/AnnotationsControls.tsx

```diff
@@ -70,6 +70,16 @@ export default function AnnotationsControls({
 
     return (
         <div className="bp-AnnotationsControls">
+            <AnnotationsButton
+                className="bp-AnnotationsControls-exitBtn"
+                data-resin-target="exit"
+                data-testid="bp-AnnotationsControls-exitBtn"
+                mode={AnnotationMode.NONE}
+                onClick={handleModeClick}
+                title={__('exit_annotations')}
+            >
+                Exit
```

Comment:
We'll want to translate this string, as well.

Reply:
I've convinced design to adopt an icon instead. We'll just need to translate the tooltip/label